### ISSUE TITLE
after installing Marvel plugin i get blank screen when browsing to the plugin

### ISSUE THREAD

**Issue by guybartal** [elastic/elasticsearch#13630]
Hi, 

I've manually installed Marvel plugin on Elastic Search from:
https://download.elasticsearch.org/elasticsearch/marvel/marvel-latest.zip
with this command "bin/plugin -i marvel -u file://PATH_TO_MARVEL_ZIP_FILE"

after the install finished successfully i browsed to http://localhost:9200/_plugin/marvel/  
and got blank screen...

**Comment by diliondani** [comment]
Same here, but the install ends with this messages:
NOTE: Unable to verify checksum for downloaded plugin (unable to find .sha1 or .md5 file to verify)
ERROR: Could not find plugin descriptor 'plugin-descriptor.properties' in plugin zip
Same error with "plugin install elasticsearch/marvel/latest"
running on windows 10 elasticsearch 2.0.0 beta 2.

**Comment by diliondani** [comment]
OK found the reason here https://discuss.elastic.co/t/marvel-won-t-install-on-elasticsearch-2-0-beta1/29110/2 Patience my friend...

**Comment by dadoonet** [comment]
Yes. Marvel is not ready yet for elasticsearch 2.0.
Stay tuned :)

**Comment by guybartal** [comment]
but i have installed Marvel on Elastic 1.7

**Comment by dadoonet** [comment]
Can you open the developer console?
Anything in browser logs?

**Comment by guybartal** [comment]
![capture](https://cloud.githubusercontent.com/assets/3179107/9980986/ee410122-5fb6-11e5-91f3-28a6e119bada.PNG)
![yml](https://cloud.githubusercontent.com/assets/3179107/9980985/ee3e1354-5fb6-11e5-9451-f4df34b60bdd.jpg)

**Comment by dadoonet** [comment]
The URL here is Kibana and not Marvel.

**Comment by guybartal** [comment]
oops, posted screenshots in wrong issue :)
i have another issue in which i can't browse kibana from remote server, only local browsing is working... i've checked and there are no FWs, but i should open this in a new Issue.

sorry, i'll get the dev console tomorrow for Marvel...

**Comment by dadoonet** [comment]
If you want to open an issue about Kibana, do it in Kibana project.

Better is to ask questions on discuss.elastic.co

**Comment by guybartal** [comment]
I've check again and while browsing http://localhost/_plugin/marvel/
I get blank page with no content. And dev console is empty also. 
Btw, I've restarted my elastic server service after installing the plugin.

**Comment by dadoonet** [comment]
Any chance you could look at the developper console and see which http calls are blocked if any?

**Comment by flyfy1** [comment]
1. I think Marvel is now supported since I can find it [on the elastic search site](https://www.elastic.co/products/marvel)
2. I'm getting white screen I think it's because I don't have `Kibana` installed.. and I found it quite hard to install
3. It turns out the only thing I want was the `ElasticSearch Sense` from Marvel. So I found [this Chrome Plugin](https://chrome.google.com/webstore/detail/sense-beta/lhjgkmllcaadmopgmanpapmpjgmfcfig?hl=en), and it is quite helpful.
